README.md

```diff
@@ -1 +1,13 @@
-# data-structures
\ No newline at end of file
+# data-structures
+
+Data structure assignments by Jared Scarr and Daniel Zwelling.
+
+Includes the following:
+
+Linked List
+Doubly Linked List
+Stack
+Deque
+BinHeap
+Priority Queue
+Graph
```

Comment:
Include actual descriptions of the data structures in your repo, not just that they exist.